python/interpret_community/mimic/mimic_explainer.py

```diff
@@ -240,6 +252,10 @@ def __init__(self, model, initialization_examples, explainable_model, explainabl
         if not all(isinstance(categorical_feature, int) for categorical_feature in categorical_features):
             categorical_features = initialization_examples.get_column_indexes(self.features, categorical_features)
 
+        # Featurize any timestamp columns
+        # TODO: more sophisticated featurization
+        self._timestamp_featurizer = initialization_examples.timestamp_featurizer()
```

Comment:
We're sure that this is something we want to run every time someone calls the mimic explainer?

Reply:
yep this was a long-standing work item, should be in other explainers eventually as well